App/views/index.erb

```diff
@@ -1,9 +1,31 @@
-<h1>Tortoise Tourism</h1>
+<style>
+body {
+  background-image: url("home-background.jpg");
+  background-repeat: no-repeat;
+  background-size: 100% ;
+}
 
-<form action='/users/new' method='get'>
-  <input type='submit' value='Sign Up'>
-</form>
-<form action='/sessions/new' method='get'>
-  <input type='submit' value='Sign In'>
-</form>
+.home-menu {
+  text-align: center;
+  margin: auto;
+  width: 40% ;
+  padding: 60px;
+  background: white;
+  border-radius: 25px;
+  margin-top: 16% ;
 
+}
+
+</style>
+
+<div class="home-menu">
+  <h1 style="padding-bottom: 40px; font-size: 70px;">Tortoise Tourism</h1>
+  <div class="btn-group mt-10">
+    <form action='/users/new' method='get'>
+      <input type='submit' class="btn btn-success btn-lg mr-3" value='Sign Up'>
```

Comment:
Can you try m-2 or m-3 on both buttons so that there is equal padding around both? (instead of mr-3 on just the left)

Reply:
done - thanks!